src/core/Akka.Remote.Tests/Akka.Remote.Tests.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\contrib\cluster\Akka.DistributedData\Akka.DistributedData.csproj" />
```

Comment:
Isn't this a circular reference?

Reply:
Can we reproduce this error without using that? Like via a custom serializer that doesn't exist?

Reply:
It shouldn't be

Reply:
It is - it depends on Akka.Cluster. Remove this.